packages/amplify_storage_s3/example/README.md

```diff
@@ -1,3 +1,54 @@
 # amplify_storage_s3_example
 
 Example app for the Amplify Flutter Storage plugin with AWS S3 provider.
+
```

Comment:
view this on https://github.com/ragingsquirrel3/amplify-flutter/blob/chore-storage-integ-tests/packages/amplify_storage_s3/example/README.md to make it slightly easier to read.